S23/sandeep-1-kumar/LowerCasing.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Locale;
+
+public class LowerCasing {
+
+    public static void main(String[] args)
+    {
+
+    }
```

Comment:
would it be better if we can avoid this extra main method?

Reply:
Yes ! I have removed it.

Reply:
great!